sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTestCommon.java

```diff
@@ -106,6 +110,62 @@ void testSizes() throws Exception {
     assertThat("Wrong estimated size", estimatedSize, greaterThan(AVERAGE_DOC_SIZE * numDocs));
   }
 
+  void testSplit5x6x() throws Exception {
```

Comment:
You need to re-architecture the tests: 
1. ok to introduce testSplit5x6x in TestCommon because ES5 and ES6 split the same and not ES2
2. ok to introduce an IT version of this test for the same reason
3. Change ESTestCommon.testSplit5x6x : it should be exactly the same as previous ES5Test.testSplit modulo the final keywords you were right to add. And UTests in ES5 and ES6 should just call     elasticsearchIOTestCommon.testSplit5x6x(); not create the index.
4. Change ESTestCommon.testITSplit5x6x: it should be exactly the same as previous ES5Test.testSplitVolume module the final keywords you were right to add.